### ISSUE TITLE
Unable to pause the stream

### ISSUE THREAD

**Issue by Moodyloaf** [orakaro/rainbowstream#297]
Expected result: The stream should pause on using the 'p' command.

Actual result: The stream continues even after using the 'p' command.

Steps to reproduce: 
1) Install rainbowstream through pip.
2) Run rainbowstream.
3) Pause stream with the 'p' command and wait until refresh timer expires.

Environment: Fedora 32 x64, bash 5.0.17

**Comment by johnpiers** [comment]
Yes, the same thing with my installation. It's not really a big deal for me, I never pause the stream anyway but although it says that it's paused it carries on regardless. Arch Linux x86_64:  zsh 5.8 (x86_64-pc-linux-gnu) Tilix

**Comment by Moodyloaf** [comment]
@johnpiers Ordinarily, that would be fine for me too, except the behaviour where a tweet I'm in the middle of composing is sent prematurely and incomplete when a stream update is displayed.

**Comment by johnpiers** [comment]
@Moodyloaf True, didn't really consider that. Then I guess that would most definitely be an irritant. :-)

**Comment by bitcoinmeetups** [comment]
This happened to me also.

**Comment by anonimoconiglio** [comment]
I'm having the same problem, is there any way to set this pause on the config file?

**Comment by fluoroom** [comment]
This is still happening to me and yes it's irritant

**Comment by ralmcruz** [comment]
I even tried to edit HEARTBEAT_TIMEOUT to 1 and STREAM_DELAY to 9999 but nothing works.
